padertorch/contrib/examples/speech_enhancement/mask_estimator/evaluate.py

```diff
@@ -45,20 +51,12 @@ def config():
 
 
 def get_checkpoint_path():
-    storage_root = os.environ.get('STORAGE_ROOT')
-    if storage_root is None:
+    if 'STORAGE_ROOT' not in os.environ:
         raise EnvironmentError(
-            'You have to specify an STORAGE_ROOT '
-            'environmental variable see getting_started'
-        )
-    elif not Path(storage_root).exists():
-        raise FileNotFoundError(
-            'You have to specify an existing STORAGE_ROOT '
-            'environmental variable see getting_started.\n'
-            f'Got: {storage_root}'
+            'You have to specify an STORAGE_ROOT environmental variable'
```

Comment:
By the way, "an" is wrong here, it should be "a" ;)
Or the word "existing" might be missing